applications/asset_tracker_v2/src/cloud/cloud_codec/json_protocol_names.h

```diff
@@ -81,6 +81,16 @@
 #define DATA_AGPS_REQUEST_TAC           "area"
 #define DATA_AGPS_REQUEST_TYPES         "types"
 
+#define DATA_AGPS_REQUEST_TYPE_UTC_PARAMETERS		 1
+#define DATA_AGPS_REQUEST_TYPE_EPHEMERIDES		 2
+#define DATA_AGPS_REQUEST_TYPE_ALMANAC			 3
+#define DATA_AGPS_REQUEST_TYPE_KLOBUCHAR_CORRECTION	 4
+#define DATA_AGPS_REQUEST_TYPE_NEQUICK_CORRECTION	 5
```

Comment:
```suggestion
#define DATA_AGPS_REQUEST_TYPE_NEQUICK_CORRECTION	 5
```
NEQUICK is not supported in the modem.

Reply:
... so the application should not request it at all.

Reply:
Support for NeQuick is (at least mostly) in place, although it's not currently used by the modem. It's even supported in the GNSS RPC API. Modem doesn't ask for it, so the application should not request it from the cloud either.

Reply:
I don't like have non-functional code in the codebase, but this is not a blocker.